ckanext/ckanext-apicatalog_scheming/ckanext/apicatalog_scheming/plugin.py

```diff
@@ -24,14 +28,48 @@
 
 _LOCALE_ALIASES = {'en_GB': 'en'}
 
-
-class Apicatalog_SchemingPlugin(plugins.SingletonPlugin):
+def ensure_translated(s):
+    ts = type(s)
+    if ts == six.text_type:
+        return s
+    elif ts == str:
+        return six.text_type(s)
+    elif ts == dict:
+        language = i18n.get_lang()
+        return ensure_translated(s.get(language, u""))
+
+
+def get_translated(data_dict, field, language=None):
+    translated = data_dict.get('%s_translated' % field) or data_dict.get(field)
+    if isinstance(translated, dict):
+        language = language or i18n.get_lang()
+        if language in translated:
+            return translated[language] or data_dict.get(field)
+        dialects = [tl for tl in translated if tl.startswith(language) or language.startswith(tl)]
+        if dialects:
+            return translated[dialects[0]] or data_dict.get(field)
+    return data_dict.get(field)
```

Comment:
Are these copied from ckanext-apicatalog_ui and added as validators instead of helpers? Why? I don't see them used anywhere in this PR.

Reply:
Removed these